tests/system/responses/test_exception_async_responses.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import struct
+from functools import partial
+
+from ..validators import validate_response_mbap
+from umodbus.client import tcp
+
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.mark.parametrize('function_code, quantity', [
+    (1, 0),
+    (2, 0),
+    (3, 0),
+    (4, 0),
+    (1, 0x07D0 + 1),
+    (2, 0x07D0 + 1),
+    (3, 0x007D + 1),
+    (4, 0x007D + 1),
+])
+async def test_request_returning_invalid_data_value_error(async_tcp_streams, mbap, function_code,
+                                                    quantity):
```

Comment:
Nitpick: Indent of this quantity argument does not line up with the rest of the argument

Reply:
Of course. Done!